### PR TITLE
add privacy-policy/index.html

### PR DESCRIPTION
**Pull request by Ivylyu**

### CODE REVIEW

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -132,7 +132,7 @@ export default class Footer extends Component {
 					</a>
 					<a className={'footer__link'} href={'https://marketplace.atlassian.com/apps/1218380/zenuml-sequence-diagram'} target="_blank">&nbsp;Confluence Plugin&nbsp;</a>
 					<a className={'footer__link'} href={'https://chrome.google.com/webstore/detail/web-sequence/kcpganeflmhffnlofpdmcjklmdpbbmef'} target='_blank'>&nbsp;Chrome Extension&nbsp;</a>
-					<a className={'footer__link'} href={'/privacy-policy/index.html'} target='_blank'>&nbsp;Privacy Policy&nbsp;</a>
+					<a className={'footer__link'} href={'privacy-policy/index.html'} target='_blank'>&nbsp;Privacy Policy&nbsp;</a>
```
Comment (mwpeng) [inline src/components/Footer.jsx]: @Ivylyu   link of EULA has been removed because we need to add other links. 
please add the link back to footer

**Comment by Ivylyu** [comment]
I have deleted the duplicate EULA folder, but the links of EULA and Privacy-policy are still invalid in my local environment, maybe because the pages are stored in the folder not published on the Internet?

**Comment by mwpeng** [comment]
@MrCoder gulpfile.js line: 36-37
any reason not to put privacy-policy inside src/.

@Ivylyu you should follow gulpfile.js line: 36-37 to copy over EULA file.

**Comment by mwpeng** [comment]
@MrCoder gulpfile.js line: 36-37
any reason not to put privacy-policy inside src/.

**Comment by Ivylyu** [comment]
I copy the EULA file but it still not fix the issue

**Comment by mwpeng** [comment]
> I copy the EULA file but it still not fix the issue

You should run `yarn serve ./app` after `yarn build`

**Comment by Ivylyu** [comment]
> > I copy the EULA file but it still not fix the issue
> 
> You should run `yarn serve ./app` after `yarn build`

I used `npm run build` but there is an error to do with babel, so I checked the  `webpack.config.js` file but there is no loader config as the StackOverflow suggested where I should add something

**Comment by mwpeng** [comment]
> > > I copy the EULA file but it still not fix the issue
> > 
> > 
> > You should run `yarn serve ./app` after `yarn build`
> 
> I used `npm run build` but there is an error to do with babel, so I checked the `webpack.config.js` file but there is no loader config as the StackOverflow suggested where I should add something

you don't need npm, just run `yarn build` then `yarn serve ./app`.  
let me know your result.

**Comment by Ivylyu** [comment]
> > > > I copy the EULA file but it still not fix the issue
> > > 
> > > 
> > > You should run `yarn serve ./app` after `yarn build`
> > 
> > 
> > I used `npm run build` but there is an error to do with babel, so I checked the `webpack.config.js` file but there is no loader config as the StackOverflow suggested where I should add something
> 
> you don't need npm, just run `yarn build` then `yarn serve ./app`.
> let me know your result.

Actually, I did `yarn build' then 'yarn serve ./app', the same issue occurred as I used `npm run build` 
I thought 'yarn build' is the same as 'npm run build', is it?

**Comment by mwpeng** [comment]
> > > > > I copy the EULA file but it still not fix the issue
> > > > 
> > > > 
> > > > You should run `yarn serve ./app` after `yarn build`
> > > 
> > > 
> > > I used `npm run build` but there is an error to do with babel, so I checked the `webpack.config.js` file but there is no loader config as the StackOverflow suggested where I should add something
> > 
> > 
> > you don't need npm, just run `yarn build` then `yarn serve ./app`.
> > let me know your result.
> 
> Actually, I did `yarn build' then 'yarn serve ./app', the same issue occurred as I used `npm run build`
> I thought 'yarn build' is the same as 'npm run build', is it?

what's your nodejs version.  I am using 10.

**Pull request merged**